docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -166,248 +166,170 @@ You need to perform the following steps:
 
 1. Login to [quay.io](https://quay.io/) to access the YugabyteDB private registry using the user name and password provided in the secret YAML file. To find the `auth` field, use `base64 -d` to decode the data inside the `yaml` file twice. In this field, the user name and password are separated by a colon. For example, `yugabyte+<user-name>:ZQ66Z9C1K6AHD5A9VU28B06Q7N0AXZAQSR`.
 
-   ```sh
-   docker login -u "your_yugabyte_username" -p "yugabyte_provided_password" quay.io
-   
-   docker search yugabytedb # You should see images
-   ```
-
-1. Fetch the YugabyteDB Helm chart on your desktop (install Helm on your desktop). Since the images in the `values.yaml` file may vary depending on the version, you need to specify the version you want to pull and push, as follows:
-
-   ```sh
-   helm repo add yugabytedb https://charts.yugabyte.com 
-   helm repo update 
-   helm fetch yugabytedb/yugaware - - version= {{ version }}
-   tar zxvf yugaware-{{ version }}.tgz
-   cd yugaware
-   cat values.yaml
-   ```
-
-   ```properties
-   image:
-   commonRegistry: ""
-    	repository: **quay.io/yugabyte/yugaware**
-    	tag: **{{ version.build }}**
-   pullPolicy: IfNotPresent
-   pullSecret: yugabyte-k8s-pull-secret
-   thirdparty-deps:
-   	registry: quay.io
-   	tag: **latest**
-   	name: **yugabyte/thirdparty-deps** 
-   prometheus:
-   	registry: ""
-   	tag:  **{{ version.prometheus }}**
-   	name: **prom/prometheus**
-   nginx:
-   	registry: ""
-   	tag: **{{ version.nginx }}**
-   	name: nginx
-   ```
+  ```sh
+  docker login -u "your_yugabyte_username" -p "yugabyte_provided_password" quay.
+  docker search quay.io/yugabyte
+  ```
+
+1. Install Helm to fetch the YugabyteDB Anywhere Helm chart on your desktop. Follow to see [Helm Installation](https://helm.sh/docs/intro/install/).
+
+1. Add Yugabytedb Anywhere Helm chart repo and check the versions in the Helm repo.
+
+  ```sh
+  helm repo add yugabytedb https://charts.yugabyte.com
+  helm repo update
+  helm search repo yugabytedb/yugaware -l
+  ```
+
+  ```sh
+  NAME                            CHART VERSION   APP VERSION     DESCRIPTION
+  yugabytedb/yugaware             2.17.0          2.17.0.0-b24    YugaWare is YugaByte Database's Orchestration a...
+  yugabytedb/yugaware             2.16.1          2.16.1.0-b50    YugabyteDB Anywhere provides deployment, orches...
+  yugabytedb/yugaware             2.16.0          2.16.0.1-b7     YugaWare is YugaByte Database's Orchestration a...
+  yugabytedb/yugaware             2.15.3          2.15.3.3-b1     YugaWare is YugaByte Database's Orchestration a...
+  yugabytedb/yugaware             2.15.2          2.15.2.1-b1     YugaWare is YugaByte Database's Orchestration a...
+  ```
+
+1. Fetch the YugabyteDB Anywhere Helm chart tar. Since the images in the `values.yaml` file may vary depending on the version, you need to specify the version you want to pull and push, as follows:
```

Comment:
It would be better to use `helm show values` instead of download the tar and cat values, like suggested in https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#customize-yugabytedb-anywhere

Also, you can use the shortcodes to indicate the version instead of writing it in the commands. See: 
https://github.com/yugabyte/yugabyte-db/blob/master/docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

Search for yb-version to understand how we are using that shortcode in the commands. The output can remain as it is, but at least the command can use the version shortcode.